app/jobs/analytics/identify_user_job.rb

```diff
@@ -0,0 +1,15 @@
+module Analytics
+  class IdentifyUserJob < AnalyticsJob
+
+    def payload(params)
+      {
+        user_id: params['owner']['id'],
+        traits: {
+          email: params['email'] || "",
+          user: params['user'],
+          account: params['account_id'] || ""
```

Comment:
please add the account type, I would like to know whether the owner is a user or an org to correlate whether we should get rid of the user plan